grafana-dashboards/legacy/Kepler-Exporter-Dashboard-openshift-carbon-usa.json

```diff
@@ -78,9 +78,7 @@
       "id": 12,
       "options": {
         "reduceOptions": {
-          "calcs": [
-            "lastNotNull"
-          ],
```

Comment:
These are autogenerated / exported files, so can we leave all the dashboards out since it is tedious to format these by hand.  
If we have a `make target` that auto-formatted, these , I am fine with it.